rskj-core/src/main/java/co/rsk/rpc/ModuleDescription.java

```diff
@@ -80,6 +85,14 @@ public boolean methodIsInModule(String methodName) {
         return true;
     }
 
+    public int getTimeout() {
+        return timeout;
+    }
+
+    public Map<String, Integer> getMethodTimeoutMap() {
```

Comment:
I'd create a `getMethodTimeout(String methodName)` method instead of this one. In this way we would hide the implementation using the map and also it would simplify the usage. 

Additionally we could create a method getMethodTimeoutOrDefault() to simplify even more to get the timeout.

Reply:
Done